cpp/src/parquet/file_reader.h

```diff
@@ -44,7 +44,8 @@ class PARQUET_EXPORT RowGroupReader {
   // An implementation of the Contents class is defined in the .cc file
   struct Contents {
     virtual ~Contents() {}
-    virtual std::unique_ptr<PageReader> GetColumnPageReader(int i) = 0;
+    virtual std::unique_ptr<PageReader> GetColumnPageReader(int i,
```

Comment:
I didn't see how the file reader deal with this new parameter. Is it intended for the caller to pass a good buffer_size?